proto/common.proto

```diff
@@ -36,6 +36,10 @@ enum MarginContract {
   ALL_SPOTS = 0;  // ALL
   SOL_SPOT = 1;   // SOL
   USDC_SPOT = 2;  // USDC
+  MSOL_SPOT = 3;  // MSOL
```

Comment:
I think we should phase these out too, get rid of these enums and use Strings... will make easier to implement some kind of an easier way to support new markets/contracts without immediate need to update the SDK. What do you think?

Reply:
yeah, its a good idea, we can create a ticket for this.